driver-core/src/main/java/com/datastax/driver/core/RollingCount.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import io.netty.util.internal.chmv8.LongAdderV8;
```

Comment:
This is marked as an internal API, but it looks like it is pulled literally from JDK8 and put into Netty's package structure.  Is the idea that we use it because it happens to be there and if it were ever removed from Netty we'd simply pull it in ourselves?